app/controllers/api/projects.php

```diff
@@ -77,6 +77,9 @@
         }
 
         $projectId = ($projectId == 'unique()') ? $dbForConsole->getId() : $projectId;
+        if($projectId === 'console') {
```

Comment:
Could you double-check if this is as string elsewhere, or do we have const for it?

Reply:
we are using similar thing in other places, so I don't think we have a constant for this
https://github.com/appwrite/appwrite/blob/9e5436f80d88f62b48c6c2c9d87dd31169a8058a/app/init.php#L742

Reply:
Awesome, thanks.